packages/sdk/README.md

```diff
@@ -0,0 +1 @@
+## VNF SDK NodeJs
```

Comment:
can you add something here on
1. link to documentation?
2. How to test?
3. how to contribute (copy from an existing repo)

FYI can be done in follow on PR

Reply:
@sloops77 I already requested those details to be added in README file in the next version of the SDK. I think, it's better to publish the SDK asap, and add update the README in the next version.